docu/files/ModuleExtensibility.html.md

```diff
@@ -31,15 +30,15 @@ Available templates:
 4. uiExtensibilityPlugin - Plug-in module example
 ```
 
-When you view the generated files, you will see that they exhibit some new features compared to a simple Builder module:
+Notice that the generated files exhibit some new features compared to a simple Builder module:
  - Metadata in the **module.json** file declares this module (**extensiblemodule**) as `extensible : nested`. The `nested` value indicates that plug-ins can be nested in the predefined slots of the host template.
- - The **index.html** file includes the **builder_plugin.js** file, which is a dedicated JavaScript API to enable integration with plug-ins.
+ - The **index.html** file includes the **builder_plugin.js** file which is a dedicated JavaScript API to enable integration with plug-ins.
```

Comment:
- The **index.html** file includes the **builder_plugin.js** file, which is a dedicated JavaScript API to enable integration with plug-ins.